### PR TITLE
chore(versions): add addons and update all packages

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

versions.js has a stable set of versions.
It doesn't support common react addons that may used accross talend project.

**What is the chosen solution to this problem?**

let's add addons used in TFD and update all current packages in UI

**Please check if the PR fulfills these requirements**
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

### CODE REVIEW

File: packages/components/src/VirtualizedList/ListGrid/__snapshots__/ListGrid.test.js.snap [hunk 1]
```diff
@@ -48,6 +48,7 @@ exports[`ListGrid should render no-rows component 1`] = `
       noRowsRenderer={[Function]}
       onRowsRendered={[Function]}
       onScroll={[Function]}
+      onScrollbarPresenceChange={[Function]}
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/ListGrid/__snapshots__/ListGrid.test.js.snap]: this is due to react-virtualized update

File: packages/components/src/VirtualizedList/ListTable/__snapshots__/ListTable.test.js.snap [hunk 2]
```diff
@@ -109,6 +109,7 @@ exports[`ListGrid should render no-rows component 1`] = `
         noRowsRenderer={[Function]}
         onRowsRendered={[Function]}
         onScroll={[Function]}
+        onScrollbarPresenceChange={[Function]}
```
Comment (jmfrancois) [inline packages/components/src/VirtualizedList/ListTable/__snapshots__/ListTable.test.js.snap]: this is due to react-virtualized update

File: version.js [hunk 3]
```diff
@@ -17,6 +17,27 @@ program
 const REACT_VERSION = '15.5.4';
 const JEST_VERSION = '20.0.3';
 
+const ADDONS = {
```
Comment (jsomsanith) [inline version.js]: This list doesn't seem to be used

File: version.js [hunk 4]
```diff
@@ -241,12 +241,12 @@ files.forEach((ppath) => {
...
-			}
-			//fs.unlink(yarnLock);
-		}
+		// const yarnLock = path.join(path.dirname(ppath), 'yarn.lock');
+		// if (fs.existsSync(yarnLock)) {
+		// 	if (!program.quite) {
+		// 		console.log(`delete ${yarnLock}`);
+		// 	}
+		// 	fs.unlink(yarnLock);
+		// }
```
Comment (jsomsanith) [inline version.js]: Please remove that, you have git history to get it back if needed

File: package.json [hunk 5]
```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "postinstall": "lerna clean --yes && lerna exec -- yarn --pure-lockfile --no-progress --mutex network && ./copylibs.js",
-    "update-versions": "./version.js",
+    "update-versions": "./version.js && lerna exec -- yarn && ./copylibs.js",
```
Comment (jmfrancois) [inline package.json]: this is to update the yarn.lock

**Comment by jmfrancois** [comment]
seems to be a screenshot issue, when you look at the live demo we still have italic + light grey

**Comment by jmfrancois** [comment]
Now I'm sure: This is because we set the 2 last browser and firefox 50 is not part of this.
So the :placeholder is supported by firefox and we have no -moz prefix in the generated css

**Pull request merged**